config/locales/jobs.yml

```diff
@@ -21,8 +21,8 @@ en:
       how_to_accept_job_applications_guide:
         description_desktop: Find out the simplest way to get candidates to apply for your jobs.
         description_mobile: Get guidance around how to accept vacancies
-        link_text: View article
-        title: How to accept job applications on Teaching Vacancies
+        link_text: Find out how to use the Teaching Vacancies application form 
```

Comment:
is it possible to add a full stop after the link, without the full stop being part of the link?

Reply:
Done!